Makefile

```diff
@@ -473,6 +473,21 @@ clean-operator: ## Clean Operator
 	oc delete awsfederatedaccountaccess --all -n ${NAMESPACE}
 	oc delete awsfederatedrole --all -n ${NAMESPACE}
 
+.PHONY: prow-ci-predeploy
+prow-ci-predeploy: predeploy-aws-account-operator deploy-aws-account-operator-credentials create-ou-map
+	@ls deploy/*.yaml | grep -v operator.yaml | xargs -L1 oc apply -f
+
 .PHONY: prow-ci-deploy
 prow-ci-deploy: ## Triggers prow-ci build and deploy operator bash script
```

Comment:
is this still needed or are you just keeping it around until you can update the make target that our prow pipeline calls?

Reply:
Yes, keeping it so that prow pipeline can trigger pipeline.
I will add a similar operating target with more appropriate name. And when this PR is merged, change the target in prow configuration.
Target name that I will use:- `prow-ci-entrypoint`